lib/npg_warehouse/loader/pacbio/run.pm

```diff
@@ -280,8 +280,9 @@ sub _ccs_info {
   my $ccs_all  = $self->_slurp_reports($reports);
 
   my %ccs;
-  if (scalar keys %{$ccs_all} > 0 && defined $ccs_all->{'HiFi Yield (bp)'}) {
-    $ccs{'hifi_read_bases'} = $ccs_all->{'HiFi Yield (bp)'};
+  if (scalar keys %{$ccs_all} > 0 && defined $ccs_all->{'HiFi Reads'}) {
+    $ccs{'hifi_read_bases'} = defined $ccs_all->{'HiFi reads yield'} ?
```

Comment:
It's probably a minor point, but this will create the 'HiFi reads yield' key if it's not in `$ccs_all`. That could be an issue if you check `exists` on it anywhere else.

Reply:
exists isn't used anywhere related to this.